packages/core-js/internals/fix-regexp-well-known-symbol-logic.js

```diff
@@ -16,13 +16,17 @@ module.exports = function (KEY, length, exec) {
     return ''[KEY](O) != 7;
   })) {
```

Comment:
Are you sure that this test also covers all engines without internal execution of `.exec` method? Maybe makes sense extend it?